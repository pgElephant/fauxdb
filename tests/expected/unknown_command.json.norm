{
  "code": 59,
  "codeName": "CommandNotFound",
  "errmsg": "no such command: unknownCommand",
  "ok": 0
}
