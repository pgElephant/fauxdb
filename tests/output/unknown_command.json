{
  ok: 1,
  isWritablePrimary: true,
  ismaster: true,
  maxBsonObjectSize: 16777216,
  maxMessageSizeBytes: 48000000,
  maxWriteBatchSize: 100000,
  localTime: ISODate('2025-09-01T21:12:42.537Z'),
  minWireVersion: 8,
  maxWireVersion: 8,
  msg: 'isdbgrid',
  hosts: [ 'localhost:27018' ]
}
