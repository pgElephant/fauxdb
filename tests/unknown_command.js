db.runCommand({unknownCommand: 1})
