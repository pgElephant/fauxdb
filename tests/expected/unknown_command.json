{
  "ok": 0,
  "code": 59,
  "codeName": "CommandNotFound",
  "errmsg": "no such command: unknownCommand"
}
